app/src/main/java/org/p2p/wallet/newsend/repository/SendServiceRepository.kt

```diff
@@ -0,0 +1,15 @@
+package org.p2p.wallet.newsend.repository
+
+import java.math.BigInteger
+import org.p2p.core.crypto.Base64String
+import org.p2p.wallet.newsend.api.GenerateTransactionResponse
+
+interface SendServiceRepository {
```

Comment:
is not used in DI + should return domain entites

Reply:
yeap, not ready yet. 
Will do in a separate PR